### PR TITLE
Update malayalam.md

### PR DESCRIPTION
**Pull request by sujik18**

### CODE REVIEW

File: living/malayalam.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
 # Web Language: Malayalam
 
-Additional names:
```
Comment (thunderpoot) [inline living/malayalam.md:3]: Why was this section removed?

File: living/malayalam.md [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-- മലയാളം
-- 
-
 News:
-- 
-- 
+- https://www.manoramaonline.com/
```
Comment (thunderpoot) [inline living/malayalam.md:4]: Descriptions for these links would be helpful, if you're able to provide them

File: living/malayalam.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
 Additional Information:
 - ISO-639-3 code: mal
 - https://en.wikipedia.org/wiki/ISO_639:mal
-
+- Unicode block: U+0D00–U+0D7F
 
 Scripts:
-- <a href="https://en.wikipedia.org/wiki/ISO_15924">ISO 15924 Mlym</a> Malayalam
-- 
+- <a href="https://en.wikipedia.org/wiki/ISO_15924">ISO 15924 Mlym</a> Malayalam
```
Comment (thunderpoot) [inline living/malayalam.md:40]: Small nitpick, a stray space was added to the end of this line

File: living/malayalam.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
 
 Scripts:
-- <a href="https://en.wikipedia.org/wiki/ISO_15924">ISO 15924 Mlym</a> Malayalam
-- 
+- <a href="https://en.wikipedia.org/wiki/ISO_15924">ISO 15924 Mlym</a> Malayalam 
 
 Thank you to these people who have helped create this document:
-- 
-- 
+- <a href="https://github.com/sujik18">Sujith Kanakkassery</a>
```
Comment (thunderpoot) [inline living/malayalam.md:43]: Please use Markdown format for this:
```md
[Sujith Kanakkassery](https://github.com/sujik18)
```

**Review by thunderpoot — CHANGES_REQUESTED** [review]
Hi @sujik18, thank you very much for your contributions.  Please see the review notes for this PR.

File: living/malayalam.md (hunk 1 above)
Comment (sujik18) [reply living/malayalam.md:3]: I thought there were no additional names for the Malayalam language, so I removed that field. I will add it again if it is necessary.

Comment (thunderpoot) [reply living/malayalam.md:3]: The name that was removed is `മലയാളം`

Comment (thunderpoot) [reply living/malayalam.md:3]: This is useful if anyone wants to look up a language using a native name in the repository search, such as https://github.com/search?q=repo%3Acommoncrawl%2Fweb-languages%20%E0%B4%AE%E0%B4%B2%E0%B4%AF%E0%B4%BE%E0%B4%B3%E0%B4%82&type=code

Comment (sujik18) [reply living/malayalam.md:3]: It’s actually not an additional name; it’s just the Malayalam word written in the Malayalam script. Should I add it?”

Comment (thunderpoot) [reply living/malayalam.md:3]: Yes please!

**Review by thunderpoot — APPROVED** [review]
Thank you very much @sujik18 for your contributions, and for addressing those comments!

**Pull request merged**